src/custom/Markdown/style.tsx

```diff
@@ -0,0 +1,9 @@
+import { styled } from '@mui/material';
+
+export const StyledMarkdown = styled('a')(({ theme }) => ({
```

Comment:
style each element  separately

Reply:
each element means different tags right?

Reply:
@aabidsofi19 i have added other majorly used elements. I have kept `h` tags on there own font sizes for now